src/main/java/org/sagebionetworks/bridge/spring/controllers/AdherenceController.java

```diff
@@ -91,4 +93,38 @@ public StatusMessage updateAdherenceRecords(@PathVariable String studyId, @PathV
         
         return service.getAdherenceRecords(session.getAppId(), search);
     }
+
+    @DeleteMapping("/v5/studies/{studyId}/participants/self/adherence")
```

Comment:
We don't need a "self" endpoint... this will only be done by people testing the system.